server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -147,41 +190,58 @@ export class OutlineRegionPicker extends LitElement {
       <outline-step-view display-action="">
         <span slot="step-title">${this.localize('region-title')}</span>
         <span slot="step-description">${this.localize('region-description')}</span>
-        <span slot="step-action">
-          <paper-button
-            id="createServerButton"
-            @tap="${this._handleCreateServerTap}"
-            ?disabled="${!this._isCreateButtonEnabled(this.isServerBeingCreated, this.selectedIndex)}"
-          >
-            ${this.localize('region-setup')}
-          </paper-button>
-        </span>
         <div class="card-content" id="cityContainer">
-          ${this.options.map((option, index) => {
-            return html` <input
-                type="radio"
-                id="card-${index}"
-                name="city"
-                value="${index}"
-                ?disabled="${!option.available}"
-                .checked="${this.selectedIndex === index}"
-                @change="${this._locationSelected}"
-              />
-              <label for="card-${index}" class="city-button">
-                <div class="flag-overlay">
-                  <img class="flag" src="${this._flagImage(option)}" />
-                </div>
-                <div class="geo-name">${getShortName(option.cloudLocation, this.localize)}</div>
-                <div class="country-name">
-                  ${option.cloudLocation.location?.countryIsRedundant()
-                    ? ''
-                    : localizeCountry(option.cloudLocation.location, this.language)}
+          <div class="card-content-row">
+            ${this.options.map((option, index) => {
+              return html` <input
+                  type="radio"
+                  id="card-${index}"
+                  name="city"
+                  value="${index}"
+                  ?disabled="${!option.available}"
+                  .checked="${this.selectedIndex === index}"
+                  @change="${this._locationSelected}"
+                />
+                <label for="card-${index}" class="city-button">
+                  <div class="flag-overlay">
+                    <img class="flag" src="${this._flagImage(option)}" />
+                  </div>
+                  <div class="geo-name">${getShortName(option.cloudLocation, this.localize)}</div>
+                  <div class="country-name">
+                    ${option.cloudLocation.location?.countryIsRedundant()
+                      ? ''
+                      : localizeCountry(option.cloudLocation.location, this.language)}
+                  </div>
+                  ${option.markedBestValue
+                    ? html`<div class="best-value-label">${this.localize('region-best-value')}</div>`
+                    : ''}
+                </label>`;
+            })}
+          </div>
+          <div class="card-content-row">
```

Comment:
Does it look okay without the row if `if-messages` returns false? Or should the `if-messages` be around this div?

Reply:
Is the Dutch for me? :smile: